spec-old-api/02-integration/05-proxy/12-server_tokens_spec.lua

```diff
@@ -80,7 +80,7 @@ describe("Server Tokens", function()
 
     setup(start {
       nginx_conf    = "spec/fixtures/custom_nginx.template",
-      server_tokens = "on",
+      headers = "server_tokens",
```

Comment:
@thibaultcha what is the approach to put new tests? Should they go into both `spec` and `spec-old-api`? '
I think only in `spec` but want to confirm

Reply:
Yes, just `spec`. `old-spec` is for backards-compatibility with 0.12 and the API entity

Reply:
Thanks for the quick response!